### PR TITLE
[0.63] Update to V8 9.7

### PR DESCRIPTION
**Pull request by tudorms**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/v8-jsi/pull/98)

### CODE REVIEW

**Comment by Khalef1** [comment]
Approving, but note that Office has moved to 0.64.

**Review by Khalef1 — APPROVED** [review]

**Review by NikoAri — APPROVED** [review]
:shipit:

File: src/inspector/inspector_socket_server.cpp [hunk 1]
```diff
@@ -338,7 +338,8 @@ bool InspectorSocketServer::HasTargets() {
 
 bool InspectorSocketServer::Start() {
   state_ = ServerState::kRunning;
-  std::thread([this]() {
+  std::unique_lock<std::mutex> tcp_server_lock(mutex_tcp_server_);
+  std::thread([this, lock = std::move(tcp_server_lock)]() {
```
Comment (NikoAri) [inline src/inspector/inspector_socket_server.cpp:342]: >this [](http://example.com/codeflow?start=15&length=4)

(nit) are we ok to pass raw pointer here?

Comment (NikoAri) [inline src/inspector/inspector_socket_server.cpp:342]: >lock [](http://example.com/codeflow?start=21&length=4)

So this is locked even before thread has a chance to execute. Should this be scoped down to start of execution?

Comment (NikoAri) [reply src/inspector/inspector_socket_server.cpp:342]: If such extended lock is indeed an expected behavior, would you mind adding brief comment explaining it?

Comment (tudorms) [reply src/inspector/inspector_socket_server.cpp:342]: It feels safer to lock before; don't know what would happen if we get through Start, then Stop, then the thread gets scheduled now touching deleted memory.

FWIW, in real-life this won't really matter; it's an issue in unit tests that quickly bring up and tear down runtimes (see 5610714 in Office).

Comment (NikoAri) [reply src/inspector/inspector_socket_server.cpp:342]: I see. If it's not too difficult, please make a comment in code about it.

**Pull request merged**